models/extension.go

```diff
@@ -6,56 +6,46 @@ import (
 	"strings"
 )
 
-// Extension holds a collection of specification extensions
-type Extensions = map[string]interface{}
-
-// UnmarshalExtensions unmarshals only specification extensions.
-func UnmarshalExtensions(data []byte) (Extensions, error) {
-	var everything map[string]interface{}
-
-	err := json.Unmarshal(data, &everything)
-	if err != nil {
+// ExtensionsFromJSON obtains AsyncAPI extensions from a JSON object.
+func ExtensionsFromJSON(data []byte) (map[string]json.RawMessage, error) {
```

Comment:
I dont see any test for this method. We should have at least one test for it :)

Reply:
You're right! Added!